tests/unit/lms/services/lti_outcomes_test.py

```diff
@@ -195,6 +220,19 @@ def make_response(cls, score, include_score, include_status, status_code):
             }
         )
 
+    @classmethod
+    def make_malformed_response(cls):
+        return xmltodict.unparse(
+            {
+                # "imsx_POXEnvelopeResponse" is the expected key
```

Comment:
Can you add a note here that this malformed response is "inspired" by Blackbaud. On that note, has anyone gotten in touch with Blackbaud to alert them to the issue? Have we looked at working around it yet?